common/men_at_arms_types/wc_cultural_maa_types.txt

```diff
@@ -1273,21 +1273,22 @@ siren = {
 snow_hunter = { 
 	type = skirmishers
 	
-	damage = 12
-	toughness = 14
-	pursuit = 0
+	damage = 14
+	toughness = 16
+	pursuit = 5
 	screen = 10
 	
 	terrain_bonus = {
-		forest = { damage = 6 toughness = 3 }
-		taiga = { damage = 12 toughness = 3 pursuit = 6 screen = 6 }
-		hills = { damage = 6 toughness = 3 }
+		forest = { damage = 3 toughness = 3 }
+		taiga = { damage = 6 toughness = 6 pursuit = 6 screen = 6 }
+		hills = { damage = 3 toughness = 3 }
+		desert = { damage = -5 toughness = -5 pursuit = -5 screen = -5 }
 	}
 	
 	winter_bonus = {
-		mild_winter = { damage = 5 toughness = 2 pursuit = 2 screen = 2 }
-		normal_winter = { damage = 6 toughness = 3 pursuit = 3 screen = 3 }
-		harsh_winter = { damage = 7 toughness = 4 pursuit = 4 screen = 4 }
+		mild_winter = { damage = 6 toughness = 6 pursuit = 6 screen = 6 }
+		normal_winter = { damage = 5 toughness = 5 pursuit = 5 screen = 5 }
+		harsh_winter = { damage = 4 toughness = 4 pursuit = 4 screen = 4 }
```

Comment:
This pretty much makes them weaker most of the time, doesn't it? Their damage output is always lower, significantly lower in taiga.

Reply:
Even in good terrain basic skirmishers also always have a smaller damage stat

Reply:
For comparison: We have Blademasters that have a damage value of 22 or 32 depending on the terrain, and we have Brutes with 17/26. With this change, you're making these into the second-lowest damage skirmishers, except if they fight in winter in their favourite terrain.

Reply:
I dont understand the issue, there will always be a worst and a best, but that is also your perspective based on which stats you think are more useful. They are just as balanced as anyone else.

Reply:
But why make these changes in the first place? They are vastly weaker anywhere now.

Reply:
Didn't the numbers we had in the previous PR (and which are merged already) not look good?

Reply:
I will resolve